docs/src/devdocs/elements.md

```diff
@@ -25,9 +25,12 @@ Ferrite.get_coordinate_eltype(::Ferrite.AbstractGrid)
 Ferrite.get_coordinate_eltype(::Node)
 Ferrite.toglobal
 Ferrite.sortface
+Ferrite.sortface_fast
 Ferrite.sortedge
+Ferrite.sortedge_fast
```

Comment:
```suggestion
Ferrite.sortface
Ferrite.sortface_fast
Ferrite.sortedge
Ferrite.sortedge_fast
Ferrite.sortfacet_fast
```

These docstrings seems a bit out of date after #914 but I can update them in a separate PR.

Reply:
I am pretty sure that we should not use `sortfacet_fast` right now. This will break queries on 4D simplices (which is still on my todo list) in a very nasty way.

Reply:
> 4D simplices (which is still on my todo list)

But please don't, that will probably break a lot of other assumptions too 😆

Reply:
When I developed the prototype the main problem was that Tensors.jl just works up to dim=3. Other than that (for linear ansatz) the generalization is straight forward for most of the codebase.